ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-12877: Error when create a Script task from Designer

## Solution
- The missing getAdminUser function was added and Error Message is not on console anymore.

## How to Test
TEST 1
- Open NEW Script from Designer Page in Assets section
- Create and Save Script, after saving Modal Window Should hide.

TEST 2

- Create New Project
- In MY PROJECTS section OPEN a project
- Add an ASSET
- Select SCRIPT
- Create and Save Script, after saving Modal Window Should hide.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-12877

## Code Review Checklist
ci:deploy
ci:next
